### PR TITLE
[W4.11e][W17-4]Bernard Teo Zhi Yi

### PR DESCRIPTION
**Pull request by btzy**
Add tests for isAnyNull() method in Utils.java.

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,33 @@ private void assertAreUnique(Object... objects) {
...
+        assertAnyNull(null, 1, 2);
+        assertAnyNull(2, 1, null);
+    }
+
+    private void assertAnyNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNoneNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:73]: It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value.
You could use the assertFalse and assertTrue inline as there are no conversions needed unlike in the elementsAreUnique method.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 2]
```diff
@@ -43,4 +43,33 @@ private void assertAreUnique(Object... objects) {
...
+        assertNoneNull("abc");
+
+        // no objects null
+        assertNoneNull("abc", "ab", "a");
+        assertNoneNull(1, 2);
+
+        // at least one object null
+        assertAnyNull(null, null, null);
+        assertAnyNull(null, 1, 2);
+        assertAnyNull(2, 1, null);
```
Comment (parvathy-sudhir-pillai) [inline test/java/seedu/addressbook/common/UtilsTest.java:65]: Commendable effort coming up with these test cases

**Review by parvathy-sudhir-pillai — COMMENTED** [review]
Overall, good effort @btzy. See my comments for suggestions. Please close this PR after reading comments.

**Comment by btzy** [comment]
Thank you.  Closing this pull request.

**Pull request closed without merging**